doc/network-ng-kickstart.md

```diff
@@ -0,0 +1,60 @@
+# Refactoring YaST Network - Kickstart Meeting
+
+Since a long time, the YaST Team has been discussing about the possibility of
+starting a sustained effort to refactor and rewrite several parts of the YaST
+Network module. On 14th March 2019 a meeting was finally held to define the
+goals of such effort and, to some extend, the strategy to follow.
+
+Since the relationship between YaST and `wicked` was considered a central point
+for the topic, both Marius and Rubén (developers of Wicked at that point in
+time) attended the meeting.
+
+## Status
+
+Currently YaST manages configurations using traditional Linux commands and
+files. That means it does not communicate directly with `wicked`. They work
+together somehow only because both understand those "legacy" formats and
+commands.
+
+YaST does not communicate with Network Manager in any way.
```

Comment:
well, maybe I would say here, that yast only start/restart service, but does not configure it in any way and neither read anything from network manager beside systemd status.